### ISSUE TITLE
[lightbox-media-viewer - with carousel]: QA: Screen reader not announcing the change in slide numbers

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8913]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2521
- DPO consulting engineer: N/A

-----

### Description

Page Element:
Previous and Next buttons in the carousel

Issue Description:
When we activate the Previous/Next buttons, visually the slides are changing. But Screen reader does not announce anything. Hence the user is not aware of this behavior.

Expected:
Screen reader should announce the change instantaneously when the previous/next buttons are activated. . For example: 1/5... 2/5... etc.

User Impact:
If the screen reader fails to identify a dynamic content change, it affects users who are blind as they may not know about the change in the page.

### Component(s) impacted

lightbox-media-viewer - with carousel

### Browser

Safari, Firefox

### Carbon for IBM.com version

canary

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/8757/iframe.html?id=components-lightbox-media-viewer--with-carousel

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by gopinathan-n** [comment]
@andy-blum @kennylam Issue is fixed in Firefox_JAWS.

Issue is not fixed in Voiceover. In iPhone_Voiceover, when we activate previous/next buttons, Voiceover announces the aria-live twice. e.g. "slide 1 of 5 showing 0 items. slide 1 of 5 showing 1 items". This is happening for all the slides.

**Comment by gopinathan-n** [comment]
Screen Reader is expected to announce the aria-live once with the correct value

**Comment by Rasubra8** [comment]
Now the following issue is observed in both jaws FF and Vo iPhone Safari,
1. Screen reader announces the aria-live incorrectly. e.g. when we activate Previous/next button, Screen reader announces the update related to both the slides instead of announcing the update only related to the current slide.
2. Each time when we press previous/next buttons, Screen reader repeats the aria-live announcement multiple times. 
Note: Earlier aria-live announcement was working fine in JAWS FF but now it is not.

**Comment by proeung** [comment]
Closing this issue out as the fixes have been addressed in this PR (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/9149).
